src/views/App.js

```diff
@@ -261,7 +261,7 @@ class App extends Component<AppP> {
                   to={{
                     pathname: '/',
                     search:
-                      '?s=N4IgbgpgTgzglgewHYgFwEYA0IA2CDGAhgC6IqqgwSFT4AWaIA-IQOYRKkAmAvAFbI8ATw4wAZFxKEq3HnE7QAthC5wSEALSTihDQHcEUHFzEBHAK7QhPKjgj5iAUgBMABhcBBF64BmUBIrejq5eAGyuMHQICKRIrDD4cMRCMDrEEAB0+DBgLgBikdGx8YnJqeqO4S4AwsEerIrIEEIZla7mMBqlQhrpigAOGlCEcRAwba3h1UlC3gAqEAPeAEoj7BkAHjgwG-kAInA+PtAc+BDe8vOL-eNVIXp0J96FMfIlM+XpWR9p5wDMeza02SVyWblWo02212zjyByOJyQZwuSFBNwmwKEZksUCEAH1kv0IDwAMoARQAMiAAL7YfDIHxwVhoUCkYh2NBIcw4HDYRQ0ADWjAARjQQNhTghVHEYGgANqgPFwLhoVzYRkQYyMT4QPFIQjKcUgegjJCaxgbI2EiCMQjmYgII3C+RoHyEbYQbBsVhQCCsdSc7m8kAAL2gjtQxCglmwCXdNtQuHk1CgRpghmItoSHGlzOwpGUAFUkElAzzYxm8nBNSrUFyebTFcq0FgQBqtYmAFRGk1IM04RhCK1CIm2+2O7DO8huj1e1g+v0BxP08ycI1h-xoKMxkBxjmJnDJsUVqCZxPSM5IXNWuBFktn+vB9Onqs1ss4RsgJW15zq6sdx86ToU1zWXBA8FTfMRwTEA7QdJ0XVQGcqDnBd-XSd9sA3CNt09XciH3JMzWPXcMyzS9r3zW8IGLUs6yDE9iFfACg2pABdPkRnMd0SSJfBMJAPRlWIBh6PLEBHiZOgH1Y6kgA'
+                      '?s=N4IgbgpgTgzglgewHYgFwEYA0IA2CDGAhgC6IqqgwSFT4AWaIA-ACYmFXEC8AVsngE8ISGAFIATADE4SYtAC2EFnBIQAtG2KE1AdwRQcLAGQBHAK7QBXKjgj5iEgAwSAgk4BmUBPKcw6CBFIkAHMYfDhiARgtOQkAYVFHF2D5ZAgBADpRADZHMxg1cMi1OXkABzUoQhCIMVys3LiIgScAFQhypwAlauCIDIAPHBgBiUkAETh3d2hhfAgnGTaOsrrnJJ06Wd9-QJlQoqiY-sPo1VEAZnGcxybI5c7xRx6aweHRqUnp2aR5xaQHqsbhk7gJTBYoAIAPqRMoQLgAZQAigAZEAAX2w+GQ7jgwTQoFIxFsaCQZhwOGw8hoAGtGAAjGggbBzBDKEIwNAAbVAULgLDQjmwuIghkYZzkUKQhEUzJA9GqSFFjAGcthEEYhDMxAQcvpMjQ7kIwwg2EIwWCUAgwVUpPJlJAAC9oLrUMQoBZsGFjRrULgZNQoHKYPpiJqwsJ2fjsKRFABVJARO0Ur2h6SigWoMkUzG8-loLAgEViv0AKjlCqQSpwjAEaoEcM12t12H15CNJrNFqtNrkjGxZlkcudXjQ7s9IG9JL9OADTNTUDDfo48yQUbVcHjiaX2YdIcX6ZLu9zID5mfEwrgGeTDsr1f7CDwQZjDd9IC1Or1BtQHaoXct1q2lm9rYCOrrjqak5ENO-pKvOk6huGq7rjGm4QAmSbASmCEHleR72uiAC6VLVGYxoInC+A3tgOj8sQDBYQ6Wx4nQO4EeiQA'
```

Comment:
was Mark able to give you good direction around this? I'm curious as to what y'all discussed.

Reply:
Yeah 😄 . We realized a library called `lz-string` was used to compress it. So we took the URL and using the library we decompressed the url to get what the actual URL was. Then I changed the actual URL to follow the format of having only `dataset` in the query params instead of both `datasetid` and `agentid`. Then I compressed the new URL using lz-string.